erpnext/accounts/report/financial_statements.py

```diff
@@ -258,3 +258,35 @@ def get_columns(period_list):
 		})
 
 	return columns
+
+
+def get_data_account_type(company, account_type, period_list):
+    data = {}
+    for period in period_list:
+        gl_sum = frappe.db.sql_list("""select sum(credit) - sum(debit) FROM `tabGL Entry`
+                                    where company='%s' and posting_date >= '%s' and posting_date <= '%s' and
+                                    account in ( SELECT name FROM tabAccount WHERE account_type = '%s')""" %
+                                    (company, period['from_date'], period['to_date'], account_type))
```

Comment:
Mapping values using `%` instead of passing values to the sql function can lead to injection, as string escaping isn't performed in the former case.